crypto-ffi/src/generic/mod.rs

```diff
@@ -110,6 +110,12 @@ pub enum MlsError {
     StaleProposal,
     #[error("The received commit is deemed stale and is from an older epoch.")]
     StaleCommit,
+    /// This happens when the DS cannot flag KeyPackages as claimed or not. It this scenario, a client
+    /// requests their old KeyPackages to be deleted but one has already been claimed by another client to create a Welcome.
+    /// In that case the only solution is that the client receiving such a Welcome tries to join the group
+    /// with an External Commit instead
+    #[error("Although this Welcome seems valid, the local KeyPackage it references has already been deleted locally. Join this group with an external commit")]
+    OrphanWelcome,
```

Comment:
I suppose we need to expose on the wasm bindings as well

Reply:
Oops, of course